### ISSUE TITLE
Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator.UnitTests.AccessibilityTests

### ISSUE THREAD

**Issue by jasonmalinowski** [dotnet/roslyn#21084]
The class Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator.UnitTests.AccessibilityTests was marked internal, which meant that none of the tests were actually being discovered. Once I did that, some tests were still failing. Those tests now point to this bug.

**Comment by CyrusNajmabadi** [comment]
These tests are public now and nothing is skipped.  closing out.
